tests/ui/fmt/help-message-issue-129966-reorder-remove-format-specifier.rs

```diff
@@ -0,0 +1,14 @@
+// Test checks for the help messages in the error output/
+//@ check-run-results
+
+struct Foo(u8, u8);
+
+fn main() {
+    let f = Foo(1, 2);
+    format!("{f:?#}");
+//~^ ERROR invalid format string: unknown format identifier '?#'
```

Comment:
Problem [REV-RUSTFIX 3/4]: `run-rustfix` will reveal that `format!()` produces a value that is marked as `#[must_use]`, so we need to

```rs
let _ = format!("{f:?x}");
```